docs/admin/predeployment_check.md

```diff
@@ -289,6 +289,26 @@ commandline (including source builds), limits should be adjusted in
 [this article](https://access.redhat.com/solutions/61334) (which is a RHEL
 specific document but the instructions apply to most Linux distributions).
 
+### Memory mapped areas
+
+Low max number of per-process mapped areas (vm.max_map_count) can cause ULT
+stack allocation to fall-back from DAOS mmap()'ed way into Argobots preferred
+allocation method.
```

Comment:
I would suggest to up-level it a little bit. Something like:

"The DAOS engine heavily uses mmap(2) to access persistent memory and to allocate stacks for asynchronous request processing via Argobots. The Linux kernel imposes a maximum limit (i.e. vm.max_map_count) on the number of mmap regions that a process can create."